src/main/java/io/confluent/connect/jdbc/dialect/GenericDatabaseDialect.java

```diff
@@ -282,7 +302,9 @@ public Connection getConnection() throws SQLException, JSchException {
         }
       }
       log.info("Remote database details {}, {}", remoteDatabaseHost, remoteDatabasePort + "");
-      int forwardedPort = session.setPortForwardingL(0, remoteDatabaseHost, remoteDatabasePort);
+      localPort = remoteDatabasePort + 1;
```

Comment:
Is there some reason that you are using `remoteDatabasePort + 1` instead of `0`?